fuse/mfs/mfs_unix.go

```diff
@@ -0,0 +1,395 @@
+//go:build (linux || darwin || freebsd || netbsd || openbsd) && !nofuse
+// +build linux darwin freebsd netbsd openbsd
+// +build !nofuse
+
+package mfs
+
+import (
+	"context"
+	"io"
+	"os"
+	"sync"
+	"syscall"
+	"time"
+
+	"bazil.org/fuse"
+	"bazil.org/fuse/fs"
+
+	dag "github.com/ipfs/boxo/ipld/merkledag"
+	ft "github.com/ipfs/boxo/ipld/unixfs"
+	"github.com/ipfs/boxo/mfs"
+	"github.com/ipfs/kubo/core"
+)
+
+// FUSE filesystem mounted at /mfs.
+type FileSystem struct {
+	root Dir
+}
+
+// Get filesystem root.
+func (fs *FileSystem) Root() (fs.Node, error) {
+	return &fs.root, nil
+}
+
+// FUSE Adapter for MFS directories.
+type Dir struct {
+	mfsDir *mfs.Directory
+}
+
+// Directory attributes (stat).
+func (dir *Dir) Attr(ctx context.Context, attr *fuse.Attr) error {
+	attr.Mode = os.FileMode(os.ModeDir | 0755)
+	attr.Size = 4096
+	attr.Blocks = 8
```

Comment:
What is the reasoning for using these magic numbers? Could we link to already existing consts instead?